examples/python/README.md

```diff
@@ -4,6 +4,11 @@ You can find Python language examples in this folder.
 
 The full details of all Looker API endpoints are listed in Looker Docs: [Version 3.1](https://docs.looker.com/reference/api-and-integration/api-reference/v3.1), [Version 4.0](https://docs.looker.com/reference/api-and-integration/api-reference/v4.0)
 
+## Notes
+- Many example scripts include the following statement `import sdk_exceptions`, which refers to this file [sdk_exceptions.py](sdk_exceptions.py). You could either uncomment the statement or download `sdk_exceptions.py` and put it in the same folder with your current script. 
+
+
+
```

Comment:
interesting - it make sense that people are probably just copy/pasting an example or snippets from an example into their own pre-configured environment (where they maybe installed the other imports from pypi) rather than within the context of cloning this repo and executing the script with Pipfile.

@josephaxisa I'm inclined to say that no standalone example within this folder should import any other module in this folder in able to better support what is probably this common use-case. If you agree, @lanlooker instead of making this change could you delete `sdk_examples.py` and just copy/paste those error classes into each individual script? Or honestly, they could all be replaced with `raise Exception` (or maybe a more suitable stdlib one from https://docs.python.org/3/library/exceptions.html but don't spend more than a few minutes trying to find a replacement, just go with `Exception`). I don't think we need to be in the business of enforcing best patterns around specific exception handling with these examples.

The exception to the "don't import other /examples/python modules" might be if an example is a package (e.g. a folder like the current `/examples/python/{hackathon_app,lookersdk-flask} examples. Then folks should realize they need that whole folder/package structure

Reply:
Agreed. I think making examples self contained makes perfect sense

Reply:
I see, I'll review and make changes to all of the scripts currently using the sdk_exceptions (I think most of them were written by @josephaxisa, I'll tag you so you could review them before merging)